homeassistant/components/season/sensor.py

```diff
@@ -53,28 +57,26 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Display the current season."""
-    if None in (hass.config.latitude, hass.config.longitude):
```

Comment:
`hass.config.latitude` and `hass.config.longitude` are marked as compulsory in the config type hint, so there is not need to check for `None`.